docs/usage/configuration/config_documentation.md

```diff
@@ -2993,10 +2993,17 @@ Options for each entry include:
 
         For the default provider, the following settings are available:
 
-       * subject_claim: name of the claim containing a unique identifier
+       * `subject_claim`: name of the claim containing a unique identifier
          for the user. Defaults to 'sub', which OpenID Connect
          compliant providers should provide.
 
+       * `picture_claim`: name of the claim containing an url for the user's profile picture.
+         Defaults to 'picture', which OpenID Connect compliant providers should provide
+         and has to refer to a direct image file such as PNG, JPEG, or GIF image file.
+         
+         Currently only supported in server configurations where media repository runs within
+         the Synapse process.
```

Comment:
I think we need to be a bit more explicit about single-process vs multiple workers configurations here.
```suggestion
         Currently only supported in monolithic (single-process) server configurations
         where media the repository runs within the Synapse process.
```